apps/api-extractor-model/src/items/ApiDocumentedItem.ts

```diff
@@ -47,7 +46,21 @@ export class ApiDocumentedItem extends ApiItem {
     const documentedJson: IApiDocumentedItemJson = jsonObject as IApiDocumentedItemJson;
 
     if (documentedJson.docComment) {
-      const tsdocParser: tsdoc.TSDocParser = new tsdoc.TSDocParser(AedocDefinitions.tsdocConfiguration);
+      const tsdocConfiguration: tsdoc.TSDocConfiguration = new tsdoc.TSDocConfiguration();
```

Comment:
This is a performance concern, since it will initialize a new `TSDocConfiguration` for every single doc comment.

Instead, we should construct it once and share it.  The `DeserializerContext` could be used to store it:

https://github.com/microsoft/rushstack/blob/a30cdf5b3caed87c63ee55f2c91dc1409fa7d29b/apps/api-extractor-model/src/model/ApiPackage.ts#L160-L168

Reply:
Good call - `DeserializerContext` now stores the `TSDocConfiguration` and non-standard tags now configured in `ApiPackage.loadFromJsonFile()`